lib/Dialect/TfheRustBool/IR/TfheRustBoolOps.td

```diff
@@ -71,4 +73,19 @@ def MuxOp : TfheRustBool_Op<"mux", [
   let results = (outs TfheRustBool_Encrypted:$output);
 }
 
+
+// ToDo: Change to an only tensor type?
```

Comment:
just to make sure, did you want to merge the TfheRustBool files with this PR or the follow-up #842?

Reply:
All of the TfheRustBool files are done in 842